### PR TITLE
Fix #3199 by honoring alpha

### LINKED ISSUE #3199: Input not registering using IE11
So I have a little project set up which utilizes the built in "click" functionality, It works fine on firefox, chrome, opera, android browser, android chrome, android firefox and IE10, but when trying to use it in IE11 it stops regestering clicks and I cant figure out why.

There are no errors being reportedand, the only console output is "Pixi.js 4.1.0 - WebGL - http://www.pixijs.com/"

### PR DESCRIPTION
**Pull request by englercj**
Proposed to fix #3199, as an alternative to #3941.

Using this test code:

```js
'use strict';

const app = new PIXI.Application({
    backgroundColor: 0x006600,
    width: 512,
    height: 512,
    view: document.getElementById('view'),
});

const graphic = new PIXI.Graphics()
    .beginFill(0xff0000)
    .drawCircle(256, 256, 200);

graphic.filters = [new PIXI.filters.NoiseFilter()];

app.stage.addChild(graphic);
```

Produces this output:

image

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
I tested this out and adjusting the `noise` property doesn't seem to do anything. I believe this is suppose to be a float from 0 to 1 of the intensity of the noise. For instance:

noise

**Comment by englercj** [comment]
I used it as a seed, not as intensity, so you could do this:

```
app.ticker.add(() => {
    graphic.filters[0].noise = Math.random();
});
```

Which will result in:

**Comment by bigtimebuddy** [comment]
It's just that this is a breaking change. That `noise` setter currently is the amount of noise, not the randomness/seed. I would think that should be a separate property.

**Comment by englercj** [comment]
Alrighty, I'll try to fix it up Soon.

**Comment by englercj** [comment]
Updated, now adjusting the `noise` property looks like:

adjust-noise

And I added a `seed` property, which when adjusted gives you some actual noise:

adjust-seed-and-noise

**Comment by bigtimebuddy** [comment]
Wow, so awesome!

File: src/filters/noise/NoiseFilter.js [hunk 1]
```diff
@@ -29,21 +29,37 @@ export default class NoiseFilter extends core.Filter
         );
 
         this.noise = 0.5;
```
Comment (bigtimebuddy) [inline src/filters/noise/NoiseFilter.js]: One very small thing that would be nice is to allow a single constructor argument for the noise value with a default for `0.5`. 

```js
// Current, non-default value
const filter = new PIXI.filters.NoiseFilter();
filter.noise = 0.1;
displayObject.filters = [ filter ];

// With constructor arg
displayObject.filters = [ new PIXI.filters.NoiseFilter(0.1) ];
```

Comment (bigtimebuddy) [inline src/filters/noise/NoiseFilter.js]: We could do this as another PR too, but thought I'd toss it in.

**Comment by englercj** [comment]
Added ctor args for `noise` and `seed`.

**Comment by bigtimebuddy** [comment]
@englercj https://travis-ci.org/pixijs/pixi.js/builds/222841155#L345-L346

**Comment by englercj** [comment]
Damn I suck, I'll fix it when I get home :P

**Comment by englercj** [comment]
Fixed the jsdoc issue, and the branch is updating now. Then should be good to go!
